app/services/bookmark_export_service.rb

```diff
@@ -103,8 +103,8 @@ def blackboard_export
   # @return [Hash] A hash containing the export data, filename, and content type
   def brightspace_export
```

Comment:
Should we call brightspace references d2l for consistency? or vice versa?

Reply:
definitely want to rename them consistently later. Just changed as little as possible to not interfere with anyone else's work.